src/main/java/uk/gov/ons/ctp/response/casesvc/domain/repository/CaseGroupRepository.java

```diff
@@ -147,4 +149,22 @@ List<CaseAction> findByCollectionExerciseIdAndActiveEnrolment(
   @Query("DELETE FROM CaseGroup cg WHERE cg.collectionExerciseId = :collectionExerciseId")
   int deleteCaseGroupsByCollectionExerciseId(
       @Param("collectionExerciseId") UUID collectionExerciseID);
+
+  /**
+   * To find Reporting Units by party UUID and survey UUID
+   *
+   * @param partyId the UUID of the Party
+   * @param surveyId the UUID of the survey
+   * @return list of ReportingUnitDTO
+   */
+  @Query(
+      value =
+          "SELECT new uk.gov.ons.ctp.response.casesvc.representation.ReportingUnitDTO"
+              + "(cg.collectionExerciseId, cg.status, c.id as caseId, c.createdDateTime, iac.iac) "
+              + "FROM CaseGroup cg, Case c "
+              + "LEFT JOIN CaseIacAudit iac ON iac.caseFK=c.casePK "
+              + "WHERE cg.caseGroupPK=c.caseGroupFK AND cg.partyId= :partyId "
+              + "AND cg.surveyId IN :surveyId ORDER BY c.createdDateTime")
+  List<ReportingUnitDTO> findReportingUnitsByPartyAndSurveyId(
+      Pageable pageable, @Param("partyId") UUID partyId, @Param("surveyId") UUID surveyId);
```

Comment:
See comment below, potentially we _will_ need to handle pagination in this query but the current use cases don't require it. Would a basic `LIMIT` clause be better? There doesn't seem to be any performance gains between that and the `fetch first 12 rows only` that JPA uses.